x-pack/solutions/observability/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/badge/sync_badge.tsx

```diff
@@ -26,6 +26,10 @@ const ASYNC_LABEL = i18n.translate('xpack.apm.transactionDetails.syncBadgeAsync'
   defaultMessage: 'async',
 });
 
+const TOOLTIP_CONTENT = i18n.translate('xpack.apm.transactionDetails.syncBadgeTooltip', {
```

Comment:
My preference is to _not_ use constants for translations, but keep them inline with the components. Having them together makes it easier for me to see "Indicates whther the span" in the UI then search for it in the code and go directly to the place it's used, rather than having to jump through constant definitions.

In this case it's in the same file so it's just a nitpick and not blocking.